### DISCUSSION TITLE
Where should I add the path to the LSP of a new language in LazyVim ???

### DISCUSSION THREAD

**Discussion by RosaUlisses** [LazyVim/LazyVim#2085] [Q&A]
I started to use LazyVim recenctly and I am quite disoriented. I had intalled the haskell LSP because I want to write some haskell code, however I do not know where should I configure the LSP in the LazyVim.

Could someone help me ?

Observation: If possible I want to do that for other languages too ...

**Comment by abeldekat** [comment]
Hi @RosaUlisses,

The lsp is described [here](https://www.lazyvim.org/plugins/lsp#nvim-lspconfig).

In the extras.lang you will find several lsp configurations. For example: [python](https://www.lazyvim.org/extras/lang/python#nvim-lspconfig).

**Reply by LeeeSe** [reply]
Can I add functions from extras to my own config file? I don't want to modify lazyvim's source code directly, but I want to use extras' handy plugin switches!
